src/game/AI/ScriptDevAI/scripts/outland/tempest_keep/botanica/botanica.cpp

```diff
@@ -19,20 +19,87 @@
 
 enum BotanicaActions
 {
+    BOTANICA_TRIGGER_BLOODWARDER_PROTECTOR_01,
+    BOTANICA_TRIGGER_BLOODWARDER_PROTECTOR_02,
+    BOTANICA_TRIGGER_BLOODWARDER_PROTECTOR_03,
+    BOTANICA_TRIGGER_BLOODWARDER_PROTECTOR_04,
     BOTANICA_ACTION_MAX,
 };
 
 instance_botanica::instance_botanica(Map* map) : ScriptedInstance(map)
 {
-    /*
-    auto posCheck = [](Unit const* unit) -> bool { return unit->GetPositionY() > 124.5f; };
-    auto successEvent = [&]()
+    // trigger 1 -1.2472258, Y:87.49708, Z:-5.6631246 
+    auto posCheckTrigger01 = [](Unit const* unit) -> bool { return unit->GetPositionY() > 87.147f; };
+    // trigger 2 X: -0.8652489 Y: 142.86328 Z: -5.539602 O: 1.5715766
+    auto posCheckTrigger02 = [](Unit const* unit) -> bool { return unit->GetPositionY() > 142.86f; };
+    // trigger 3 X: -0.39172745 Y: 219.03831 Z: -5.54034 O: 1.5785537
+    auto posCheckTrigger03 = [](Unit const* unit) -> bool { return unit->GetPositionY() > 219.038f; };
+    // trigger 4 X: -8.046078 Y: 288.01355 Z: 0.2993633
+    auto posCheckTrigger04 = [](Unit const* unit) -> bool { return unit->GetPositionX() > -8.046f && unit->GetPositionY() > 288.013f; };
+
+    auto successEvent01 = [&]()
+    {
+        auto m_bloodwarderGroup = instance->GetCreatures(THIRD_BLOODWARDER_STRING);
+        if (m_bloodwarderGroup != nullptr)
+        {
+            for (Creature* creature : *m_bloodwarderGroup)
+            {
+                if (!creature->IsAlive())
+                    return;
```

Comment:
Shouldn't this be `continue` instead of `return`? You don't want to skip `MoveWaypoint` for all living creatures after, do you?